File: wwwroot/js/Utilities/RequestConfig.js
function getRequestConfig(RequestCompression, RequestEncryption, ResponseCompression, ResponseEncryption) {
  return encryptData(JSON.stringify({
    'RequestCompression': RequestCompression,
    'RequestEncryption': RequestEncryption,
    'ResponseCompression': ResponseCompression,
    'ResponseEncryption': ResponseEncryption
  }))
}

File: Models/RequestConfig.cs
using System.Runtime.Serialization;

namespace Atithya_Web.Models
{
  [DataContract(Namespace = "")]
  public class RequestConfig
  {
    [DataMember]
    //0 for false and 1 for true
    public string? RequestCompression { get; set; }
    [DataMember]
    //0 for false and 1 for true
    public string? RequestEncryption { get; set; }
    [DataMember]
    //0 for false and 1 for true
    public string? ResponseCompression { get; set; }
    [DataMember]
    //0 for false and 1 for true
    public string? ResponseEncryption { get; set; }
  }
}
